doc/nvim-treesitter.txt

```diff
@@ -417,7 +417,12 @@ This is left as an exercise for the reader.
 `TSInclude`
 								*hl-TSInclude*
 For includes: `#include` in C, `use` or `extern crate` in Rust, or `require`
-in Lua
+in Lua.
+
+`TSAnnotation`
+                                                             *hl-TSAnnotation*
+For C++/Dart attributes, annotations that can be attached to the code to
+denote some kind of meta information.
```

Comment:
Could this also be used for python decorators? :thinking:

Reply:
I like the current highlighting as functions, since decorators are not annotations in that sense but indeed modify the code. And decorators are in-fact defined as functions.